### ISSUE TITLE
hyper 1.0 support

### ISSUE THREAD

**Issue by aslilac** [routerify/routerify#122]
wanted to set up a new project using hyper and routerify, but the example no longer works when using hyper 1.0

**Comment by is-it-ayush** [comment]
Yep! Had to switch to `0.14` hyper to make routerify work. Would be awesome if routerify `4.0` becomes compatible with hyper `1.0`. 🙂

**Comment by Guru901** [comment]
Hi @aslilac and @is-it-ayush , if u still want to use routerify with hyper 1.7 i just completed porting it, i m publishing it right now
u can find it here - https://crates.io/crates/routerify_ng

**Comment by Guru901** [comment]
It's very new actually I just got all the tests to pass, so if you are willing to test it out in some of your projects  that'd be great

**Comment by Guru901** [comment]
Update - I just migrated one of my own project [ripress](https://crates.io/crates/ripress) that has 26k downloads so a real thing from hyper 0.14 to hyper 1.7 using this version of routerify.
